livekit-ffi/src/server/mod.rs

```diff
@@ -82,6 +80,7 @@ pub struct FfiServer {
     config: Mutex<Option<FfiConfig>>,
     logger: &'static logger::FfiLogger,
     handle_dropped_txs: DashMap<FfiHandleId, Vec<oneshot::Sender<()>>>,
+    rpc_method_invocation_waiters: Mutex<HashMap<u64, oneshot::Sender<Result<String, RpcError>>>>,
```

Comment:
This shouldn't be inside the FfiServer, I think this should be inside FfiRoom

Reply:
done